examples/github-notification-filters/app.tsx

```diff
@@ -148,7 +154,7 @@ autocomplete<AutocompleteItem>({
       showQuerySource && querySource,
       showPrefixesSource && prefixesSource,
       showPostfixesSource && postfixesSource,
-    ].filter(Boolean);
+    ].filter(Boolean) as Array<AutocompleteSource<AutocompleteItem>>;
```

Comment:
Not every source returns items that exactly satisfy the `AutocompleteItem` contract (sometimes there's more data, sometimes there's less), but there's enough overlap so that it doesn't matter, and each source has its own template anyway.

I think I can improve types in a separate PR since this is only a demo.